app/models/public/tagged_term.rb

```diff
@@ -1,20 +0,0 @@
-module Public
-  class TaggedTerm < Public::PublicBase
-    #  Note:  Read-only All public data is managed by other apps (aact & aact-proj).
-    after_initialize :readonly!
-
```

Comment:
Make sure to check for any calls to the methods in this model. Search for the method names. Note if they are saved to variables.

If there are lots of uses of these methods then we'll want to keep the model. If the methods don't seem to be not used or used very rarely, then we can probably remove the model. Though we will have to fix what was using them.